azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -473,4 +481,13 @@ private Thread startThread(final JobRunner runner) {
     thread.start();
     return thread;
   }
+
+  private VersionSet createVersionSet(){
+    final String testJsonString1 = "{\"azkaban-base\":{\"version\":\"7.0.4\",\"path\":\"path1\","
+        + "\"state\":\"ACTIVE\"},\"azkaban-config\":{\"version\":\"9.1.1\",\"path\":\"path2\","
```

Comment:
Just want to verify. for the job, we are just emitting version number or versionInfo right?

Reply:
We are emitting only version number.